docs/Salesforce-batchsink.md

```diff
@@ -32,6 +32,12 @@ A Salesforce connected application must be created in order to get a client secr
 
 **SObject Name:** Salesforce object name to insert records into.
 
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
+
+**Connection:** Name of the connection to use. Object Names information will be provided by the connection.
+You also can use the macro function ${conn(connection-name)}.
```

Comment:
The .md has a problem. The example for SObject Name is under the Connection property now.

Reply:
Changed the sequence.